TSIS2.WebResources/Webresources/js/tsis2findingwidget.js

```diff
@@ -73,7 +73,13 @@
 
         file.onchange = function () {
             for (var i = 0; i < file.files.length; i++) {
-                fileArray.push(file.files[i].name);
+                alert("File size: " + file.files[i].size);
+                if(file.files[i].size > 10240000){
+                    alert("The maximum file size of 10 MB has been exceeded. Please choose another file.");
```

Comment:
Need help localizing this?

Reply:
Also, rather than using JS alert, consider using this for a consistent D365 UI experience https://docs.microsoft.com/en-us/powerapps/developer/model-driven-apps/clientapi/reference/xrm-navigation/openalertdialog